tests/test-team/template-mgmt-component-tests/sms/template-mgmt-create-sms-page.component.ts

```diff
@@ -228,9 +228,7 @@ test.describe('Create SMS message template Page', () => {
 
       await createSmsTemplatePage.messageTextArea.fill(templateMessage);
 
-      await createSmsTemplatePage.clickContinueButton();
-
-      await page.getByRole('button', { name: 'Continue' }).click();
```

Comment:
???

Reply:
There is actually a bug in this test.

If you update the playwright timeout and run the app in a debug terminal you'll see it. 

when browser clicks the continue button the web-app takes a few seconds to handle redirecting to a different page.

When the assertions for inputs are executed they happen on the same page before the app has had time to redirect the user and the test passes, which is wrong. 

What the test _should_ be doing (this is the same for Email). Is;

1. Loading a session with no data
2. Entering data
3. Clicking the continue button (save the data is saved)
4. Return back to CreateEmailPage 
5. Assert the inputted data is correct

Reply:
Updated to click on the back link after clicking on the Continue button